src/App.jsx

```diff
@@ -1,60 +1,127 @@
-import "./App.css"
+import "./App.css";
+import { useState } from "react";
 
 function App() {
+  const [first, setFirst] = useState("0");
+  const [second, setSecond] = useState("0");
+  const [sign, setSign] = useState("+");
+  const [result, setResult] = useState(0);
+  const [store, setStore] = useState("");
+
+  const firstUpdate = (val, res = false) => {
```

Comment:
change name, this is unclear: maybe setFirstNumber is better

unclear what val and res represent.. choose clearer names